doc/releases/release-notes-2.6.rst

```diff
@@ -159,6 +161,12 @@ Drivers and Sensors
 
 * Flash
 
+  * flash_write_protection_set() was deprecated API and become no-operation.
+    For all the driver service of flash write protection was embedded into write
+    and erase operations implementations. Driver implementation interface API
+    :c:type:`flash_api_write_protection` becomes unneeded anymore, and was
```

Comment:
I think instead of `becomes unneeded anymore`, `is not needed anymore` or `becomes unneeded` would be better.

Reply:
Actually text was incorrect.